common/src/main/java/com/wynntils/models/players/PartyModel.java

```diff
@@ -396,6 +407,13 @@ public void partyCreate() {
         McUtils.sendCommand("party create");
     }
 
+    /**
+     * Join another players party
+     */
+    public void partyJoin(String playerName) {
```

Comment:
joinParty would be a bit nicer imo

Reply:
Maybe, but I'm following a pre-existing naming pattern here for methods.